internal/jimm/juju/jimm.go

```diff
@@ -278,7 +279,7 @@ func (j *JujuManager) PrepareModelMigration(
 	modelUUID string,
 	targetControllerName string,
 	userMapping map[string]string,
-) error {
+) ([]byte, error) {
```

Comment:
Update godoc, to explain it's returning a token too. 

Side thing:
Was also just thinking, might be good to add a claim to the token for the incoming model migration, even just an id would be nice. That way we can scope our tokens per migration, because right now you could be in migration an and b, and swap the tokens which I don't think is right. Feel free to leave this though.